### PR TITLE
Un-comment the Mclock parameter check test case and ms_osd_compression_algorithm parameter to snappy

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
This PR is to un-comment and modify the ms_osd_compression_algorithm to snappy. Doing the modification due to the following bugs are fixed.
1.https://bugzilla.redhat.com/show_bug.cgi?id=2124137
2.https://bugzilla.redhat.com/show_bug.cgi?id=2215272
3.https://bugzilla.redhat.com/show_bug.cgi?id=2155380

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/rados/tier-2_rados_test_parameters.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#################################################################################
```
Comment (pdhiran) [inline suites/reef/rados/tier-2_rados_test_parameters.yaml:1]: Any reason for a New test suite? Can't we add these tests to existing suite?

**Comment by pdhiran** [comment]
Please add the pass logs for all the tests. please run the entire suite, of which the modified tests are a part of.

**Comment by SrinivasaBharath** [comment]
Reef Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XFRA4Z

File: suites/reef/rados/tier-2_rados_test_parameters.yaml (hunk 1 above)
Comment (SrinivasaBharath) [reply suites/reef/rados/tier-2_rados_test_parameters.yaml:1]: This suite I added in Quincy and not included in the reef. So I added this in Reef. The test cases are all related to checking parameters and execution will take around 5 hours.  That is the reason I kept it separate.

**Comment by SrinivasaBharath** [comment]
Quincy-Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5QR2W3

File: tests/rados/test_config_parameter_chk.py [hunk 2]
```diff
@@ -24,10 +26,18 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     osd_list = []
     mclock_profile = ["balanced", "high_recovery_ops", "high_client_ops"]
-    # To get the parameters from the ini
     rados_object = RadosOrchestrator(node=cephadm)
     mon_object = MonConfigMethods(rados_obj=rados_object)
-
+    cmd_get_ceph_ver = "ceph -v"
```
Comment (pdhiran) [inline tests/rados/test_config_parameter_chk.py:31]: This method would require code change added for every release. Can we pass the ini file from the test suite?

Comment (SrinivasaBharath) [reply tests/rados/test_config_parameter_chk.py]: Passing the ini file from the test suite.

**Comment by SrinivasaBharath** [comment]
Test execution is in progress.

**Comment by pdhiran** [comment]
Looks good. Once we have pass logs in reef or quincy, can be merged!

**Comment by SrinivasaBharath** [comment]
Quincy Pass Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-DBX0GR

**Review by pdhiran — APPROVED** [review]
Looks good!

**Pull request merged**